pages/browse/index.tsx

```diff
@@ -1,18 +1,54 @@
 import RCHead from "../../src/components/Head/RCHead"
 import Layout from "../../src/components/Layout/Layout"
 import { SITE_NAME } from "../../src/config/constants"
+import { fetchSubjects } from "../../src/server/api/browse"
+import initializePatronTokenAuth from "../../src/server/auth"
+import type { HTTPStatusCode } from "../../src/types/appTypes"
+import type { DiscoverySubjectsResponse } from "../../src/types/browseTypes"
+
+interface BrowseProps {
+  subjects: DiscoverySubjectsResponse
+  isAuthenticated: boolean
+  errorStatus?: HTTPStatusCode | null
+}
 
 /**
  * The Browse index page is responsible for fetching and displaying subject headings,
  * as well as displaying and controlling pagination and sort.
  */
-export default function BrowseIndex() {
+export default function Browse({
+  subjects,
+  isAuthenticated,
+  errorStatus = null,
+}: BrowseProps) {
   const metadataTitle = `Browse Research Catalog | ${SITE_NAME}`
 
   return (
     <>
       <RCHead metadataTitle={metadataTitle} />
-      <Layout activePage="browse"></Layout>
+      <Layout activePage="browse" isAuthenticated={isAuthenticated}></Layout>
     </>
   )
 }
+
+export async function getServerSideProps({ req, query }) {
+  const patronTokenResponse = await initializePatronTokenAuth(req.cookies)
+
+  const subjectsResponse = await fetchSubjects({
```

Comment:
Same here. let's set ourselves up for authors with a browse_type param or something.

Reply:
I meant that `getServerSideProps` could parse a browse_type param, and call fetchSubjects based on that.